python/ngraph/ops.py

```diff
@@ -202,6 +202,18 @@ def reshape(node, output_shape, input_order=None, name=None):
     return Reshape(node, AxisVector(input_order), Shape(output_shape))
 
 
+@nameable_op
+def elu(data, alpha, name=None):  # type: (NodeInput, NodeInput, str) -> Node
+    """Perform elu operation on data node element-wise with the specified alpha parameter.
+
+    :param data: One of: input node, array or scalar.
```

Comment:
We now added information on supported types, but what is this param supposed to provide in terms of the function's logic?